examples/nonlinear_elasticity.cpp

```diff
@@ -0,0 +1,868 @@
+//                       libROM MFEM Example: Nonlinear elasticity
+//
+// Compile with: make nonlinear_elasticity
+//
+// Sample runs:
+//    mpirun -np 4 nonlinear_elasticity -s 3 -rs 2 -dt 3
```

Comment:
The error in this test is over 100%
srun -n 1 -p pdebug nonlinear_elasticity -s 3 -rs 2 -dt 3
I would not expect all the sample runs to have good results with DMD, but at least one sample run for each example should work. Did you find a case that works?

Reply:
I am actually working on this, @copeland. I will post the working examples today.

Reply:
Try the following:

./dg_advection -p 3 -rp 1 -dt 0.005 -tf 4 -visit
mpirun -n 8 ./dg_euler -p 2 -rs 2 -rp 1 -o 1 -s 3 -visit
mpirun -np 8 ./heat_conduction -s 3 -a 0.5 -k 0.5 -o 4 -tf 0.7 -vs 1 -visit
mpirun -np 8 ./nonlinear_elasticity -s 2 -rs 1 -dt 0.01 -tf 5 -visit

Reply:
Yes, these work. I wonder why some cases do not work for DMD, but we don't have to answer that now. There should be documentation on sample runs that work for DMD.

Reply:
There was a small bug with dg_advection. The results are a lot better now after my latest commit. It should work well with the default "srun -n 8 -p pdebug dg_advection". The other examples should be unaffected.

Reply:
Added example run documentation to the top of each file.